cloud_AWS/terraform/module/variables.tf

```diff
@@ -101,3 +101,23 @@ variable "external_id" {
   type        = string
   default     = ""
 }
+
+variable bucket_arn_list {
```

Comment:
If I understand correctly, this variable is for internal use and not to be specified by user; if so - it should be described

Reply:
Resolved